src/main/java/httpserver/Main.java

```diff
@@ -18,10 +21,11 @@ public static void main(String[] args) {
             try {
                 SocketServer socketServer = new HTTPSocketServer(new ServerSocket(5000));
                 String path = "/Users/molliestephenson/Java/Server/cob_spec/public";
+                String resources = "/Users/molliestephenson/Java/Server/HTTPServer/src/main/resources/form";
```

Comment:
I've changed this in the next pull request.  

@fabientownsend the java syntax I'm using is `getClass().getClassLoader().getResources("/form")` if you ever need it in java!